data/announcements/scheduled.yaml

```
@@ -116,3 +116,16 @@ announcements:
   message: |
     5 days of incredible opportunites to collaborate, learn + share with the entire community!<br />
     May 16 - 20, 2022.
+
+- name: Kubecon 2022 NA
+  startTime: 2022-10-15T00:00:00 # Added in https://github.com/kubernetes/website/pull/36107 
+  endTime: 2022-10-28T18:00:00
```

I suggest to start the Monday of the week before KubeCon and to end at midnight the Saturday after KubeCon

```suggestion
  startTime: 2022-10-17T00:00:00 # Added in https://github.com/kubernetes/website/pull/36107 
  endTime: 2022-10-29T00:00:00
```

---

Right...sounds good to me